projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/bees/solitary/reed.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "甘蔗蜜蜂",
```

```suggestion
    "name": "芦苇蜜蜂",
```
这个应该更正为芦苇蜜蜂，注意en_us中的也应当修改，不要改漏

---

en_us为什么要改？那不是原文吗

---

淦打错了，指的是zh_cn（）